app/test_main.py

```diff
@@ -1,3 +1,42 @@
+import pytest
+from typing import Any
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, human_age",
+    [
+        (-1, -1, [0, 0]),
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 23, [1, 1]),
```

Comment:
in this case you check one only for a dog and the other one only for a cat, but you should check them for both